### ISSUE TITLE
Balls fall right through red gears

### ISSUE THREAD

**Issue by chrisjj** [jessecrossen/ttsim#27]
Actually, behind them.

E.g. https://jessecrossen.github.io/ttsim/#s=13,17&z=48&cc=7.5&cr=8.5&t=1&pt=8&sp=1&sc=0&b=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA0AAAASCAYAAACAa1QyAAAAAXNSR0IArs4c6QAAAPBJREFUOE+1kzEOwiAUhh+Lc0dHOIMHaAcP0gbvUS5C0t7DgdHJeAMYnYwnKPraggivcZKkSfn5P35e+2DGmKltW8BnGAZomgaMMRBGriulgAUITdY6EIJHILxo7UDKRScgy4JRCOGt/Z4XUGooolYBN4pJdV0vCWwN8v7DoZbMx3H0c00RCmAKZRoNUWBygt9QMONGa/o2FCrK6kH5TxDx7X8nbUFKqSlf6/s+dkbXdclPW5yMc15A1lkQXIB2GiSXRV4BpV1NtROuk0kBpCC8PiSEZgSpgfdt7sYTP8C1esDueYR7dYb9zcFlq9Xf+gs7cqew2FcrKgAAAABJRU5ErkJggg==

![image](https://user-images.githubusercontent.com/6157073/153728110-045de0a9-fd33-415d-a164-ae13c904f14d.png)

This in latest Google Chrome on Windows.

**Comment by Grape-Boy** [comment]
pretty sure that's intended

**Comment by VuiMuich** [comment]
Yes, red gears are just passive linking elements and don't interact with the marbles.
